README.rst

```diff
@@ -62,3 +62,12 @@ To build a full URL use the ``full`` method like this:
 You can implement the `get_scheme(url)` and `get_hostname(url)` methods on your
 `Url` class to change your default theme and hostname from the urlman defaults
 of `'http'` and `'localhost'`, respectively.
+
+If you use Django RestFramework, you can use ``urlman.UrlManField`` to provide
```

Comment:
There should be a space between Rest and Framework, and it should be `REST` not `Rest`, I think?

Reply:
Between `pip install djangorestframework` and `from rest_framework import …` I never remember how this thing is even called, but you're of course right.